mne/channels/layout.py

```diff
@@ -145,6 +149,10 @@ def read_layout(kind, path=None, scale=True):
     -------
     layout : instance of Layout
         The layout.
+
+    See Also
+    --------
+    save
```

Comment:
The `Layout` class doesn't seem to be listed in the API reference page. It should be, right?